students/andykwon/lesson02/gridprinter.py

```diff
@@ -0,0 +1,14 @@
+def print_grid(num):
```

Comment:
Looks good!

In the future you would probably want to store all of the repeated parts in a variable name.  If your boss was like, "oh no we wanted ampersands and not plus signs" it would be tedious to go through and copy and paste new code and this could lead to errors. I'd recommend storing these and then printing them instead, next time:
```
top_line = '+' + ' - ' * int(num/2) + ' + ' + ' - ' * int(num/2) + '+'
middle_line = '| ' + ' ' * int(num/2)*3 + '|' + ' ' * int(num/2)*3 + ' |'
```